packages/atomic/src/utils/jsx-utils.tsx

```diff
@@ -5,12 +5,14 @@ export interface LocalizedStringProps {
   bindings: AnyBindings;
   key: string;
   params: Record<string, VNode | string>;
+  i18nCount?: number;
```

Comment:
IMO every parameter is already related to i18n, so I would replace this with `count`.